authenticatorbridge/src/main/java/com/bitwarden/authenticatorbridge/factory/AuthenticatorBridgeFactory.kt

```diff
@@ -0,0 +1,31 @@
+package com.bitwarden.authenticatorbridge.factory
+
+import android.content.Context
+import com.bitwarden.authenticatorbridge.manager.AuthenticatorBridgeManager
+import com.bitwarden.authenticatorbridge.manager.AuthenticatorBridgeManagerImpl
+import com.bitwarden.authenticatorbridge.manager.model.AuthenticatorBridgeConnectionType
+import com.bitwarden.authenticatorbridge.provider.SymmetricKeyStorageProvider
+
+/**
+ * Factory for supplying implementation instances of Authenticator Bridge SDK interfaces.
+ */
+class AuthenticatorBridgeFactory(
+    private val applicationContext: Context,
```

Comment:
I think we should do what you've done below and ensure that the context is only an application context:

```kotlin
class AuthenticatorBridgeFactory(
    context: Context,
) {
    private val applicationContext = context.applicationContext

```

Reply:
Done.